redis/src/main/scala/zio/redis/api/Geo.scala

```diff
@@ -45,12 +59,18 @@ trait Geo {
    *  @param members additional members
    *  @return chunk of geohashes, where value is `None` if a member is not in the set
    */
-  final def geoHash(
-    key: String,
-    member: String,
-    members: String*
-  ): ZIO[RedisExecutor, RedisError, Chunk[Option[String]]] =
-    GeoHash.run((key, (member, members.toList)))
+  final def geoHash[K: Schema, M: Schema](
+    key: K,
+    member: M,
+    members: M*
+  ): ZIO[RedisExecutor, RedisError, Chunk[Option[String]]] = {
+    val command = RedisCommand(
+      GeoHash,
+      Tuple2(ArbitraryInput[K](), NonEmptyList(ArbitraryInput[M]())),
+      ChunkOptionalMultiStringOutput
```

Comment:
I think we should consider to do arbitrary type here, too.
What do you think @mijicd ?

Reply:
I agree. Let's adjust based on usage / needs, and keep it as it is for now.